core/native/refresh/src/main/csharp/Services/IconProvider.Path.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ch.cyberduck.core;
+using System;
+
+namespace Ch.Cyberduck.Core.Refresh.Services
+{
+    public partial class IconProvider<T>
+    {
+        public T GetPath(Path path, int size)
```

Comment:
Moved from WinFormsIconProvider to base - previously used in ProgressController.